src/components/AddressDetailExplorer.js

```diff
@@ -167,26 +175,24 @@ class AddressDetailExplorer extends React.Component {
    *
    * @param {Object} queryParams URL parameters
    */
-  getHistoryData = (page) => {
-    // update query parameters
-    const query = this.pagination.obtainQueryParams();
-    if (query.page !== page) {
-      query.page = page;
-      const newURL = this.pagination.setURLParameters(query);
-      this.props.history.push(newURL);
-    }
-
-    return addressApi.getHistory(this.state.address, this.state.selectedToken, TX_COUNT, TX_COUNT * (page-1)).then((response) => {
+  getHistoryData = (lastTx, lastTs) => {
+    console.log('Downloading')
```

Comment:
Should we remove this?

Reply:
Yes we should, removed in 3301af9cb627c140dfd8019387f81792fbad7738